build.gradle

```diff
@@ -17,10 +17,10 @@
 plugins {
     id 'base'
     id 'maven-publish'
-    id 'net.researchgate.release' version '2.6.0'
-    id 'de.undercouch.download' version '4.1.1'
-    id "com.github.johnrengelman.shadow" version "5.2.0"
-    id "com.github.spotbugs" version "4.0.5"
+    id 'net.researchgate.release' version '2.8.0'
+    id 'de.undercouch.download' version '5.4.0'
+    id "com.github.johnrengelman.shadow" version "7.1.2"
```

Comment:
Shouldn't we change this to a version `8.x` (`8.1.1`)

Reply:
```suggestion
    id "com.github.johnrengelman.shadow" version "8.1.1"
```